From 63845f8ff80688cd752c4f0a031d1672e72a064c Mon Sep 17 00:00:00 2001
From: Stefan Reimer <it@startux.de>
Date: Tue, 31 Jul 2012 11:36:17 +0200
Subject: [PATCH] i18n/module-setup.sh: fixed include parsing

Due to bug in module-setup.sh in the i18n module includes from the main
keymap are not working and not added to the initrd image.

Only one quotation per line is removed. Needs to remove all.
---
 modules.d/10i18n/module-setup.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/10i18n/module-setup.sh b/modules.d/10i18n/module-setup.sh
index 6d44a69..7f15ad9 100755
--- a/modules.d/10i18n/module-setup.sh
+++ b/modules.d/10i18n/module-setup.sh
@@ -31,7 +31,7 @@ install() {
             *) cmd=grep ;;
         esac
 
-        for INCL in $($cmd "^include " $MAP | while read a a b; do echo ${a/\"/}; done); do
+        for INCL in $($cmd "^include " $MAP | while read a a b; do echo ${a//\"/}; done); do
             for FN in $(find ${kbddir}/keymaps -type f -name $INCL\*); do
                 findkeymap $FN
             done
